src/style.css

```diff
@@ -0,0 +1,154 @@
+/* #region Normalize */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+/* #endregion */
+
+.card {
+  width: 200px;
+  height: 408;
```

Comment:
do not add height with hard coded value , try to adjust with padding and margin
to each element add bottom or top margin (do not both to one element)

Reply:
understood. thanks